packages/terra-application-header/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra App Delegate
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-app-delegate.svg)](https://www.npmjs.org/package/terra-app-delegate)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The AppDelegate is an object that provides a set of functions used to facilitate communcation between Containers and
```

Comment:
Update description to: `The ApplicationHeader is a HOC that combines config and menu subcomponents, which are then placed within the header layout.`